tests/testthat/test-derive_vars_dtm.R

```diff
@@ -789,7 +782,7 @@ test_that("derive_vars_dtm Test 31: catch ignore_seconds_flag error", {
       dtc = XXSTDTC,
       ignore_seconds_flag = TRUE
     ),
-    error = TRUE
+    error = FALSE
```

Comment:
This unit test should test that an error is issued. I.e., the input data needs to be updated such that the error is thrown.